pkg/cmd/render/env.go

```diff
@@ -82,7 +81,10 @@ func getElectionTimeout(platform, arch string) (map[string]string, error) {
 }
 
 func getUnsupportedArch(platform, arch string) (map[string]string, error) {
-	if !strings.HasPrefix(arch, "s390") {
+	switch arch {
+	case "arm64":
+	case "s390x":
```

Comment:
we need to cover s390 case as well as s390x

Reply:
Why?  Plain s390 means the very old 31-bit userspace, for which RHEL has only had multilibs in previous versions and not at all in RHEL 8.  There are certainly no plans for OCP in that configuration.

Reply:
ok I have no context here to what s390 is outside of "both were possible" if you're saying that is not true and regression is not a concern then OK with me.